### PR TITLE
Added test for request and class method, added to the merchant index action as well

### PR DESCRIPTION
**Pull request by MDelarosa1993**
This PR adds the ability to create new items through the Items API, including the necessary routes, serializer formatting, and tests to ensure proper functionality and response structure.

### CODE REVIEW

File: spec/requests/api/v1/merchants_requests_spec.rb [hunk 1]
```diff
@@ -129,18 +129,24 @@
...
     end
   end
-end
 
-  # it "can return the count" do
-  #   merchant1 = Merchant.create!(name: "Brown and Sons")
-  #   merchant2 = Merchant.create!(name: "Brown and Moms")
-  #   merchant3 = Merchant.create!(name: "Brown and Dads")
+  it 'returns merchants sorted by creation date (newest first)' do
+    Merchant.create!(name: "Merchant A", created_at: 2.days.ago)
```
Comment (CCirbo) [inline spec/requests/api/v1/merchants_requests_spec.rb:134]: Good testing coverage.

File: app/models/merchant.rb [hunk 2]
```diff
@@ -1,9 +1,11 @@
 class Merchant < ApplicationRecord
     has_many :items, dependent: :destroy
     has_many :invoices, dependent: :destroy
+
+    def self.sorted_by_age
```
Comment (CCirbo) [inline app/models/merchant.rb:5]: Good use of class method!

**Comment by CCirbo** [comment]
Good job Melchor!

**Comment by jimmacur** [comment]
Great improvements! Moving the sorting logic into the model keeps the controller clean and makes the code more maintainable. The added tests provide solid coverage, ensuring everything works as expected. Nice job! 

I believe this is the branch that has the work that Lito was also working on. I think we will have some merge conflicts when his comes through, so we will have to see which code we want to use. Good to know that both were working.

**Pull request merged**